pkg/network/tracer/testutil/kafka/testdata/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+# Taken from https://github.com/conduktor/kafka-stack-docker-compose/blob/master/zk-single-kafka-single.yml
+
+version: '3'
+
+services:
+  zoo1:
+    image: confluentinc/cp-zookeeper:7.2.1
```

Comment:
These images need to be uploaded to the internal docker registry; we don't want CI jobs to fail because we run into docker's limit. You can maybe pass in the registry URL as a env var, and still fallback to docker's registry URL if that env var is not set to facilitate running this test locally.

Reply:
Currently the CI is disabled
Furthermore, even with internal registry is won't work as the kitchen tests do not have docker credentials nor logged in to aws account
I've opened a ticked for the platforms team, but it seems like they don't have a solution just yet...

Reply:
https://dd.slack.com/archives/C03FVCR2ATU/p1668437701726479